about.php

```diff
@@ -39,17 +39,17 @@
 					<h3>Socials</h3>
 				  <p>No student organisation is complete without the friends &amp; family that one makes throughout their tenure. We develop our students as people, not just academically. The aspect of building relationships is just as important as any academic teaching because the relationships that students forge when taking part in our social events can improve their academic life by engaging with like-minded and driven individuals.</p>
 					<h3>Talks</h3>
-				  <p>?</p>
+				  <p>At UNT Robotics, we invite companies and individuals out to talk about their expertise. We have had a number of esteemed talks by <a href="https://www.linkedin.com/in/david-evenden/">David Evenden</a> of <a href="https://www.standardusercyber.com/">Standard User Cyber Security</a>, engineers from <a href="https://www.bellflight.com/">Bell Flight</a>, <a href="https://www.linkedin.com/in/maddenmengwasser/">Madden Mengwasser</a> from <a href="https://www.se.com/ww/en/"></a> and many more. If you would like to come speak at one of our events, <a href="./contact">contact us!</a></p>

```

Comment:
the very last link seems to be empty, also we shouldn't use `./contact` -- all links should be absolute on the site so it should be `/contact`
I also can't approve the PR until you resolve the merge conflicts, because after you commit your conflict resolutions it'll take away my review :p

Reply:
bump :)